spec/requests/nonprofits_spec.rb

```diff
@@ -1,53 +1,48 @@
 # License: AGPL-3.0-or-later WITH Web-Template-Output-Additional-Permission-3.0-or-later
 require 'rails_helper'
-require 'support/contexts/shared_donation_charge_context'
 describe NonprofitsController, type: :request do
-  include_context :shared_donation_charge_context
-
+  let(:nonprofit) { create(:nonprofit_base, :with_default_billing_subscription)}
```

Comment:
Not using the `shared_donation_charge_context` reduces the amount of context that we need to understand in order to understand the spec.